Resources/Prototypes/Entities/Objects/Weapons/Melee/e_sword.yml

```diff
@@ -177,6 +177,80 @@
     sound:
       path: /Audio/Effects/unwrap.ogg
 
+- type: entity
+  name: energy dagger
+  parent: EnergySword # I don't just parent it to energy dagger because then it would be able to write on paper.
```

Comment:
just give it an edagger, being able to write on paper isnt much when the borg can just talk anyway